app/components/mig/login/wb_setting.svg

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 18.1.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
These were the raw assets from Google Drive. If this is important you might want to look at using SVGO. Seems outside the scope of this PR though.